types/testing-library__dom/pretty-dom.d.ts

```diff
@@ -1,3 +1,7 @@
 import { OptionsReceived } from 'pretty-format';
 
-export function prettyDOM(element: HTMLElement, maxLength?: number, options?: OptionsReceived): string | false;
+export function prettyDOM(
+    dom?: HTMLElement | HTMLDocument,
```

Comment:
This is too narrow. It excludes svg.

```suggestion
    dom?: Element | HTMLDocument,
```

Reply:
Thanks! Done